src/screens/IssuesScreen.tsx

```diff
@@ -1,20 +1,25 @@
 import React, { FC } from "react";
 import { StyleSheet, View } from "react-native";
-// import IssuesItem from '../components/IssuesItem';
-import Navigation from "../components/Navigation";
-// import {useGlobalContext} from '../contexts/AppContext';
-import IssuesList from "../components/IssuesList";
+import { Navigation } from "components/Navigation";
+import { IssuesList } from "../components/IssuesList";
 import { NativeStackScreenProps } from "@react-navigation/native-stack";
 import { RootStackParams } from "../types";
-import SortingModal from "../components/SortingModal";
+import { SortingModal } from "components/SortingModal";
+import { useIssuesData } from "hooks/useIssuesData";
 
 type Props = NativeStackScreenProps<RootStackParams, "Issues">;
 
 const IssuesScreen: FC<Props> = ({ navigation, route }) => {
+  const { isLoading, issuesData } = useIssuesData();
   return (
     <View style={styles.container}>
       <View style={styles.issues}>
-        <IssuesList navigation={navigation} route={route} />
+        <IssuesList
+          navigation={navigation}
```

Comment:
this prop can be changed with `onIssuePress` or something like that, it shouldn't get passed plain `navigation`